ME3Explorer/NativesEditor/Views/StateEventMapView.xaml.cs

```diff
@@ -508,7 +520,28 @@ private void AddSubstateSiblingIndex_Click(object sender, System.Windows.RoutedE
 
         private void RemoveSubstateSiblingIndex_Click(object sender, System.Windows.RoutedEventArgs e)
         {
-            RemoveSubstateSiblingIndex(SubstateStateEventSiblingIndicesListBox.SelectedIndex);
+            var button = (Button)sender;
+            RemoveSubstateSiblingIndex((int)button.Tag);
+        }
+    }
+
+    /// <summary>
+	///   Resolves a name entry for a given Pcc
+	/// </summary>
+    public class NameConverter : IMultiValueConverter
+    {
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (values[0] is IMEPackage pcc && values[1] is int index)
+            {
+                return $"{pcc.GetNameEntry(index)}_{values[2]}";

```

Comment:
Thanks for the pull request, this all looks great! There's just one small issue:

This is not the correct way of displaying instanced names. The instance number should be decremented by 1 for display, and if it's 0, the name is not instanced. (eg. If the name is "Liara" and the instance is 1, it would be displayed as "Liara_0", and if the instance is 0, it would just be displayed as "Liara")

You can leverage the NameReference class to perform this logic for you, like so:

```suggestion
                return new NameReference(pcc.GetNameEntry(index), values[2]).Instanced;

```